sig-apps/README.md

```diff
@@ -24,7 +24,6 @@ The Chairs of the SIG run operations and processes governing the SIG.
 
 * Janet Kuo (**[@janetkuo](https://github.com/janetkuo)**), Google
 * Kenneth Owens (**[@kow3ns](https://github.com/kow3ns)**), Brex
-* Matt Farina (**[@mattfarina](https://github.com/mattfarina)**), Rancher Labs
```

Comment:
@mattfarina don't you want to move yourself below to Emeritus Leads section?

Reply:
fixed